src/modules/premium/server/procedures.ts

```
@@ -0,0 +1,72 @@
+                return null;
+            }
+
+           const product = await polarClient.products.get({
+                id: subscription.productId,
+            });
+
+            return product;
+        }),
+
+    getProducts: protectedProcedure
+        .query(async ({ }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)